Document editorial changes from January 2025. (#342)
ynput/ayon-documentation: website/docs/addon_resolve_artist.md

</div>
</div>

### Reviewable source

In case you wish to publish reviewable video per `plate`:
1. Enable the `review` toggle

> 2. Select either the appropriate track name or `[clip's media]` (will use the clip own source)

Reviewer: Same. Can't find the `review` toggle.

### Vertical synchronization of product attributes

This feature allows to publish multiple plate variant(s) per shot.

> The timeline must contain multiple tracks - as mentioned [here](#rename-timeline-track-names) one of those being the hero (driving) track.

Reviewer 2: ```suggestion
The timeline must contain multiple tracks - as mentioned [here](#rename-timeline-track-names) - one of those being the hero (driving) track.
```

Per shot, the clip on the hero track will be driving the shot creation (shot length, shot resolution...)

To publish multiple plates for a shot process as following:

1. Set up a multiple track timeline. Rename each track to determine plate variants (e.g `L01`, `L02` and `L03`)
2. Pick-up one track the be the **hero** track and set your main plate clip onto it
3. Then, for each shot, stack additional plate clip(s) on other tracks aligned with the hero/main plate clip. Clips can be shorter than the main hero track clip, but not longer.
4. (Optional) You can setup another track to contain your review material
5. Select all clips that needs to be published (with chocolate color)
6. In the creator enable vertical align and select your main track as hero track
5. Create instances by hitting "Create"

> 6. Ensure all instances are parented to the same shot, which lenth is driven from the hero clip

Reviewer 2: ```suggestion
6. Ensure all instances are parented to the same shot, whose length is driven by the hero clip.
```

</div>
</div>

### Reviewable source

> ![Create menu](assets/resolve_review_toggle.png)

Reviewer: Just checking. the attributes shown in this image don't exist in the previous screenshots.
For reference, 
![image](https://github.com/user-attachments/assets/50dc9774-fb2c-4af6-8b69-0e56c07c2f39)

Author: This is because it's an instance attribute and not a creator attribute. I've rework this and took another screenshot. 
Hopefully this is clearer now.
